### PR TITLE
Initial documentation for R9M Lite Pro

### PR DESCRIPTION
**Pull request by olliw42**
title says it. First attempt.

### CODE REVIEW

File: docs/FRSKY_R9.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
-# mLRS Documentation: Frsky R9M, R9MX #
+# mLRS Documentation: Frsky R9M, R9M Lite Pro, R9MX #
 
 ([back to main page](../README.md))
 
-The Frsky R9M transmitter module and R9MM and R9MX receivers are commercially available and hence can be a relatively easy way to get started with mLRS. However, the R9 hardware does have some limitations as detailed below which may affect some applications.
+The Frsky R9M and R9M Lite Pro transmitter modules and R9MX and R9MM receivers are commercially available and hence can be a relatively easy way to get started with mLRS. However, the R9 hardware does have some limitations as detailed below which may affect some applications.
```
Comment (jlpoltrack) [inline docs/FRSKY_R9.md:5]: Perhaps:

'The Frsky R9M, R9M Lite Pro transmitter modules and R9MX, R9MM receivers can be a relatively easy way to get started with mLRS as they are commercially available.'

File: docs/FRSKY_R9.md [hunk 2]
```diff
@@ -85,14 +85,36 @@ Connections:
...
 - Vin
 - GND
-- Pin 5 (S.Port) - Out (SBUS only)
+- Pin 5 (S.Port) - Out (SBus only)
+
+## R9M Lite Pro as Receiver ##
+
+The R9M Lite Pro transmitter module could be used as mLRS Tx module but it is not well suited for this. It can however be a great option as mLRS receiver, providing 1 W transmit power and hence excellent range in combination with a R9M Tx module.
+
+The R9M Lite Pro has no button; for binding the power cycle method is available (see [here](BINDING.md)). It also does not provide the CLI, and configuration hence must be done via the Lua script (see [here](LUA.md)).
```
Comment (jlpoltrack) [inline docs/FRSKY_R9.md:101]: Perhaps:

'The R9M Lite Pro has no button therefore the power cycle method has to be used for binding (see [here](BINDING.md)).

Nitpick - should we mention the CLI at all?  Receivers by themselves don't use the CLI for configuration.

**Comment by jlpoltrack** [comment]
Looks good - just a couple of comments.

File: docs/FRSKY_R9.md (hunk 2 above)
Comment (olliw42) [reply docs/FRSKY_R9.md]: 1. gerat thx. Done
2. uuupppssss  ... that's not nitpick but pointing to a serious nonsense :) MANY THX. removed.

**Comment by olliw42** [comment]
I think not. The R9M ones are also not there. But I agree that in discord they are not very searchable. You think we should add a section with them, maybe at the end?

**Comment by jlpoltrack** [comment]
> I think not. The R9M ones are also not there. But I agree that in discord they are not very searchable. You think we should add a section with them, maybe at the end?

How about only one image that indicates where the SWD connection is?  I can understand skipping R9M as now one can use without opening the case but R9M Lite Pro requires SWD to update.

**Comment by olliw42** [comment]
ah, valid point. I didn't wanted to do pics at this point but just get "something" out. Should this pic be located in the STLink section, or the R9M Lite Pro main section? I think when I spend the ime to do a pic for R9M Lite Pro I well can do also a pic for R9M, so it would be two pics. I have a tendency to put them into the STLink section. What do you say?

**Comment by jlpoltrack** [comment]
> ah, valid point. I didn't wanted to do pics at this point but just get "something" out. Should this pic be located in the STLink section, or the R9M Lite Pro main section? I think when I spend the ime to do a pic for R9M Lite Pro I well can do also a pic for R9M, so it would be two pics. I have a tendency to put them into the STLink section. What do you say?

STLink section makes sense to me, as that is similar to the other pages.

**Comment by olliw42** [comment]
added
also adapted the links for the other to teh elrs pages

**Comment by jlpoltrack** [comment]
Looks good to merge if you're all set :)

**Comment by olliw42** [comment]
I'm all set :)

**Pull request merged**